src/templates/inscription.rs

```diff
@@ -19,6 +19,18 @@ pub(crate) struct InscriptionHtml {
   pub(crate) timestamp: DateTime<Utc>,
 }
 
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+pub struct InscriptionDetailsJson {
+  pub address: Option<String>,
```

Comment:
```suggestion
  pub address: Option<Address>,
```

Reply:
@casey @raphjaph I couldn't figure this out. Followed the lead of what we already do with `InscriptionJson`. Can you please help?